src/Camera.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.camera.Camera" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../../../xsd/widget.xsd">
+    <name>Camera</name>
+    <description>Allows users to take and save pictures on their mendix applications</description>
```

Comment:
Description should specify that this can only happen with a webcam. Mendix applications also run on mobile.

Reply:
Done